### DISCUSSION TITLE
DNS_PROBE_FINISHED_NXDOMAIN instalation problem

### DISCUSSION THREAD

**Discussion by cloudengineer89** [nextcloud/all-in-one#3210] [Questions] [answered]
Please help me to resolve this problem.
I've gone through steps 1-4 and after I run the packages and click on the button 'Open your Nextcloud', I receive this error:
`DNS_PROBE_FINISHED_NXDOMAIN`
My server is located on the ec2.

**Comment by szaimen** [comment, marked as answer]
Hi, I've found this: https://www.google.com/search?q=DNS_PROBE_FINISHED_NXDOMAIN
